selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -258,6 +258,9 @@ private Locators() {}
   @FindBy(xpath = Locators.AUTOCOMPLETE_PROPOSAL_JAVA_DOC_POPUP)
   private WebElement autocompleteProposalJavaDocPopup;
 
+  @FindBy(xpath = Locators.ALL_TABS_XPATH)
+  private WebElement anyOpenTab;
```

Comment:
`anyOpenTab` is grammatically incorrect.
Better variant: `someOpenedTab` (http://esl.fis.edu/grammar/rules/some.htm)